ml-proto/test/memory.wasm

```diff
@@ -26,6 +27,17 @@
   (module (memory 100 1000 (segment 0 "a") (segment 2 "b") (segment 1 "c")))
   "data segment not disjoint and ordered")
 
+;; Test alignment annotation rules
+(assertinvalid (module (func (loadu.2.i8 (const.i32 0)))) "alignment too big")
```

Comment:
Too big alignment is [part of the design](https://github.com/WebAssembly/design/blob/ce509cdb9ec33fc439e389c82dffc1e641c1925d/AstSemantics.md#alignment) so this test isn't correct.